packages/ember-glimmer/tests/integration/syntax/with-test.js

```diff
@@ -292,4 +292,43 @@ moduleFor('Syntax test: Multiple {{#with as}} helpers', class extends RenderingT
     this.assertText('Los Pivots');
   }
 
+  ['@test nested {{#with}} blocks should have access to root context']() {
+    this.render(`{{name}}{{#with committer1.name as |name|}}[{{name}}{{#with committer2.name as |name|}}[{{name}}]{{/with}}{{name}}]{{/with}}{{name}}{{#with committer2.name as |name|}}[{{name}}{{#with committer1.name as |name|}}[{{name}}]{{/with}}{{name}}]{{/with}}{{name}}`,
```

Comment:
We've been using the `strip` helper to keep things readable. It removes all leading whitespace and newlines. For example, check out https://github.com/emberjs/ember.js/blob/master/packages/ember-glimmer/tests/integration/syntax/each-in-test.js#L110-L116.